doc-Managing_Infrastructure_and_Inventory/topics/Instance_Evacuation.adoc

```diff
@@ -1,4 +1,4 @@
-If a Compute node is shut down, you can _evacuate_ instances hosted on it. This is only useful if the instances use shared storage or Block Storage volumes. See https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/single/instances-and-images-guide/#section-migration-evacuation[Evacuate Instances] (in the Red Hat OpenStack Platform _Instances and Images Guide_) for details on the underlying OpenStack process.
+If a Compute node is shut down, you can _evacuate_ instances hosted on it. This is only useful if the instances use shared storage or Block Storage volumes. See https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/single/instances-and-images-guide/#section-migration-evacuation[Evacuate Instances] (from the Red Hat OpenStack Platform _Instances and Images Guide_) for details on the underlying OpenStack process.
```

Comment:
Hey Suyog,  
No worries! I wanted to check with you about it too anyway :) Please feel free to ask someone else to merge this PR while I'm away too if that's more convenient. Looking forward to hearing how the discussion goes :)
Cheers,
Dayle